components/src/components/top-button/top-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Listen, Prop, h } from '@stencil/core';
+
+// Scroll to top button.
+@Component({
+  tag: 'pulumi-top-button',
```

Comment:
Nit -- four spaces and double-quotes, and should be applied throughout. (This irritates me, too; Stencil generates these stubs with two spaces and single quotes, and there doesn't seem to be a config setting to override that, sadly. We should probably add some linting rules for it, though.)